README.md

```diff
@@ -8,3 +8,10 @@ pos
 Point of sale
 
 
+Note
+~~~~
+
+Regarding lint warning and errors on Travis you can follow the steps
```

Comment:
Isn't better a link to the general guidelines?

Reply:
In fact, we usually put this file in root: https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/CONTRIBUTING.md

Reply:
yes indeed ! but is there a part of guidelines that mentions how to use eslint ?

Reply:
If not, we should add it :wink:

Reply:
Hi. 
- I removed the last commit
- added the contributing.md file.
- proposed some text in the OCA guidelines (https://github.com/OCA/odoo-community.org/pull/43)

Is it all ok ? 

Thanks ! 

regards.